### ISSUE TITLE
Option to completely disable backdrop on Modals.

### ISSUE THREAD

**Issue by Spellhammer** [soflyy/oxygen-bugs-and-features#649]
Posted on behalf of @alriksson

**Describe the feature you'd like to see included in Oxygen.**
A toggle or control to allow disabling the Modal backdrop entirely.

**What are the use cases for this feature?**
Things like hello bars, where you want to display a Modal along an edge of the viewport but want the user to be able to continue interacting with the website.

**Comment by wplit** [comment]
Yes, very useful for site notifications, welcome notices etc that don't need to be immediately actioned or dismissed.

**Comment by wplit** [comment]
You can already do this will custom CSS.

`.oxy-modal-backdrop {
     pointer-events: none;
}

.ct-modal {
     pointer-events: auto;
}

Then give backdrop a transparent background. But having this as a built-in would be great.
